python/tvm/ir/transform.py

```diff
@@ -120,6 +120,11 @@ def current():
         """Return the current pass context."""
         return _ffi_transform_api.GetCurrentPassContext()
 
+    @staticmethod
+    def list_configs():
```

Comment:
chang this api name as well?

Reply:
Done, thanks. Do you mind merging it once CI finishes?